pkgs/development/python-modules/poetry/default.nix

```diff
@@ -29,11 +33,20 @@ let
       sha256 = "06zp695hq835rkaq6irr1ds1dp2qfzyf32v60vxpd8rcnxv319l5";
     };
   });
-  glob2 = callPackage ./glob2.nix { };
 
+  # used for tests, can't be used directly for install because
+  # it's missing a setup.py
+  githubSrc = fetchFromGitHub {
+    owner = "sdispater";
+    repo = "poetry";
+    rev = version;
+    sha256 = "004s747wkil5f00r0vjff73r6vhlrrcnfan2k7pl7gyf62wfp02a";
+  };
```

Comment:
If needs must, I guess - at least both package fetches are based on the same `version` so they shouldn't diverge.

Reply:
yes, which is why i moved the version to the the let scope...

Reply:
only way I thought i could sneak tests in... didn't want to break it

Reply:
plus, there's apparently no packages that use poetry to build... so i can't use them as a way to verify correctness :(

Reply:
the source can be used by using `format = "pyproject";`

Reply:
the raw source of poetry can't be used without a previous version of poetry, the github pull is just to get the tests.